### PR TITLE
add webtau library definition

### PR DESCRIPTION
**Pull request by MykolaGolubyev**
This is to expose WebTau (https://github.com/testingisdocumenting/webtau) for incremental end to end testing experience

### CODE REVIEW

File: webtau.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "dependencies": [
+    "org.testingisdocumenting.webtau:webtau:$v"
+  ],
+  "imports": [
+    "org.testingisdocumenting.webtau.WebTauDsl.*"
+  ],
+  "init": [
+    "org.testingisdocumenting.webtau.browser.BrowserConfig.setBrowserSameDriverInThreads(true)"
+  ],
+  "renderers": {
```
Comment (MykolaGolubyev) [inline webtau.json:16]: in a future version of webtau it would make more sense to add a catch all renderer for any class that implements PrettyPrintable

**Comment by ileasile** [comment]
Cool! Do you have an example notebook that shows this integration in action?

**Comment by MykolaGolubyev** [comment]
Attached 
[webtau-notebook.ipynb.zip](https://github.com/Kotlin/kotlin-jupyter-libraries/files/11382873/webtau-notebook.ipynb.zip)

**Comment by MykolaGolubyev** [comment]
Do I need to add anything else to proceed?

**Comment by MykolaGolubyev** [comment]
Thank you!

**Comment by ileasile** [comment]
Thank you for the contribution!

**Comment by ileasile** [comment]
@MykolaGolubyev there are some problems with the integration and the notebook
1. This element can't be found for some reason (though google.com page for me contains About link)
image
2. Rendering can fail with exception. Even if rendering of this object doesn't make sense, it's better to show toString() representation, but not throw
image

**Comment by MykolaGolubyev** [comment]
Ah interesting findings. I will need to investigate why xml parser fails on rendering. 

Will repeat my google thingy. Recently did a small demo with another site and things were working. Very curios about what may have gone wrong. 

Thank you for pointing them out. 
just in case - recorded demo here https://twitter.com/mgolubyev/status/1654133713138069504?s=46&t=tZAK1PYTTCeD6mI9LGUr8A

**Comment by ileasile** [comment]
Wow! That's very cool, thank you for sharing!

**Comment by MykolaGolubyev** [comment]
@ileasile I am not able to repro "About" issue. My notebook looks like this
image

**Comment by ileasile** [comment]
@MykolaGolubyev I've updated my comment. It looks like my browser opens google.com in German for some reason (maybe because I'm in Germany now..). Maybe it's possible to set fixed locale for a browser?

**Comment by MykolaGolubyev** [comment]
Oh wow. Will check if I can add a few options to selenium wrapper. Are these critical? Was wondering if you were building a demo or something.

**Comment by ileasile** [comment]
No, it's not critical) I just wanted to add the notebook to our examples and decided to check the integration a bit beforehand)

Also, is it stated somewhere that you actually need Chrome installed on your computer? I understand that it should be obvious, but it disappointed me a bit when I got an error trying to run the notebook in Datalore

**Pull request merged**